compiler/circle-opselector-test/README.md

```diff
@@ -0,0 +1,52 @@
+# Circle-OpSelector-test
```

Comment:
Please use same convention for name. You can see how others are named under `compiler`.

For this case, 
```suggestion
# circle-opselector-test
```
would be good.